src/test/java/org/assertj/core/presentation/StandardRepresentation_format_CompletableFuture_Test.java

```diff
@@ -38,7 +39,8 @@ public void should_format_complete_future() {
   public void should_format_failed_future() {
     CompletableFuture<Object> future = new CompletableFuture<>();
     future.completeExceptionally(new RuntimeException("some random error"));
-    assertThat(STANDARD_REPRESENTATION.toStringOf(future)).isEqualTo("CompletableFuture[Failed: java.lang.RuntimeException: some random error]");
+    assertThat(STANDARD_REPRESENTATION.toStringOf(future)).matches("CompletableFuture\\[Failed: java.lang.RuntimeException: some random error\\]\n"
```

Comment:
avoid using `\n`, I remember windows contributors having tests failing because of such usage.